.clang-format

```diff
@@ -0,0 +1,110 @@
+# This configuration file can be used to auto-format the code base.
+# Not all guidelines specified in CODING_STYLE are followed, so the
+# result MUST NOT be committed indiscriminately, but each automated
+# change should be reviewed and only the appropriate ones commited.
+#
+# To apply the coding style you can run the following command (assuming you
+# installed clang-format-6.0 on your system):
```

Comment:
```suggestion
# installed clang-format on your system):
```

Maybe remove the specific clang-format version everywhere in the extended comment? We're not committing the results indiscriminately so the specific clang-format version used shouldn't really matter that much.